lib/msal-core/README.md

```diff
@@ -18,14 +18,31 @@ Via NPM:
 Via CDN:
 
     <!-- Latest compiled and minified JavaScript -->
-    <script src="https://secure.aadcdn.microsoftonline-p.com/lib/<version>/js/msal.js"></script>
-    <script src="https://secure.aadcdn.microsoftonline-p.com/lib/<version>/js/msal.min.js"></script>
+    <script type="text/javascript" src="https://{CDN-URL}/msal.js" integrity="sha{alg}-{checksum}" crossorigin="anonymous"></script>
+    <script type="text/javascript" src="https://{CDN-URL}/msal.min.js" integrity="sha{alg}-{checksum}" crossorigin="anonymous"></script>
```

Comment:
I'm not sure there's any value in including these lines.

Reply:
taken out